src/main/java/com/google/sps/servlets/ResultsServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Map;
+import java.util.HashMap;
+import com.github.mustachejava.Mustache;
+import com.github.mustachejava.MustacheFactory;
+import com.github.mustachejava.DefaultMustacheFactory;
+
+/** Servlet that shows results. */
+@WebServlet("/results")
+public class ResultsServlet extends HttpServlet {
```

Comment:
[optional] maybe try to name this servlet something more specific to keyword results, unless you are planning on generalizing this logic to show the results for actions and books

Reply:
Well, it is serving the template that is used to render keywords, actions and books.

Reply:
Nvm then